enhancements/olm/platform-operators.md

```diff
@@ -0,0 +1,609 @@
+- As a cluster admin that installed a PO that's no longer needed, I want to be able to delete a PO without having to uninstall the cluster.
+- As a cluster admin, I want to be able to approve new versions of an individual PO package.
+- As a team that has a direct dependency on an OLM-based Operator, I want to be able to tailor my installation to meet my use case.
+- As an operator author, I want admins to be able to get my operator installed by default and automatically upgraded as the cluster upgrades so that I can provide "core" cluster functionality that looks cohesive with the rest of the cluster/payload.
+
+### Goals
+
+- Admins can select and configure POs before cluster creation.
+- Admins can install and delete a PO after cluster creation.
+- Ensure that OLM-based operators can be installed during day 0 operations.
+- Ensure that any failed PO installations result in a failed cluster installation.
+- Ensure that only AllNamespace OLM-based operators can be installed.
+- Respect any Kubernetes/OCP cluster version constraints while selecting candidate bundles.
+- Respect the "operatorframework.io/suggested-namespace" CSV annotation value.
+
+### Non-Goals
+
+- Prohibit multiple PO packages from being installed on the cluster during phase 0.
+- Provide a first-class UX for configuring POs via the install-config API during phase 0.
+- Provide a unified admin UX for installing OLM-based operators through this PO mechanism vs. the normal OLM workflow during phase 0.
+- Enable POs to block cluster upgrades during phase 0.
+- Onboard optional, CVO-based operators to OLM's management during phase 0.
+- Detach optional, CVO-based operators from the [payload lifecycle](#payload-lifecycle) during phase 0.
+- Enable all OLM-based operators to be deployed via the PO mechanism during phase 0.
+- Support any essential operator to be deployed via the PO mechanism. Future phase designs may re-evaluate this agent-based approach.
+- Support for extending the PO mechanism to cover any MAPI/CCM use cases that will enable supplementing OpenShift with platform-specific integrations during phase 0.
+- Support sourcing catalog content from any CatalogSource in the cluster during phase 0.
+- Support dependency resolution for PO packages during phase 0.
+- Support automatic upgrades for PO packages during phase 0.
+- Support more fine-tuned health checks during phase 0.
+- Require PO packages to report state through an auxiliary API (e.g. ClusterOperator, OperatorCondition, etc.) during phase 0.
+- Provide a mechanism for admins to manually approve PO package upgrades during phase 0.
+- Permit non-essential operators to be rolled back.
+
+## Terminology
+
+### Payload Lifecycle
+
+These are components that build/ship with the OCP payload as a unit. This unit includes both core, control plane components, in addition to the independent, "value" add components that all release on the same release cadence.
```

Comment:
> in addition to the independent, "value" add components that all release on the same release cadence

Is this a typo or am I misunderstanding what you mean by independent, "value" add components? The payload lifecycle is independent of the release cycles of OLM operators, which means they have (potentially) different release cadences, right?